ods_ci/tests/Tests/1300__model_registry/1302_model_registry_model_serving.robot

```diff
@@ -260,9 +267,27 @@ Remove Model Registry
     Run And Verify Command    oc delete secret modelregistry-sample-rest-credential -n ${NAMESPACE_ISTIO}
     Run And Verify Command    oc delete namespace ${NAMESPACE_MODEL-REGISTRY} --force
 
-Remove Certificates
-    [Documentation]    Remove all files from the certificates directory
-    ${files}=    List Files In Directory    ${CERTS_DIRECTORY}
+Remove Deployment Files
+    [Documentation]    Remove all files from the given directory
+    [Arguments]  ${directory}
+    ${files}=    List Files In Directory    ${directory}
     FOR    ${file}    IN    @{files}
-        Run Process    rm    -f    ${CERTS_DIRECTORY}/${file}
+        Remove Files  ${directory}/${file}
+    END
+
+Download Python Client Dependencies
+    [Documentation]  Download the model-registry package for a specific platform
+    [Arguments]  ${destination}  ${package_version}
+    ${result}=    Run Process    command=pip download --platform=manylinux2014_x86_64 --python-version=3.9 --abi=cp39 --only-binary=:all: --dest=${destination} ${package_version}    # robocop: disable:line-too-long
```

Comment:
> pip download

Can you explain how `pip download` will work in an airgap environment without specifying the index URL?

Reply:
Are you uploading the file to a folder and then "downloading" from there? Is this the way?

Reply:
Also, why set platform and abi? Why not use the defaults?
* Defaults to the platform of the running system
* If not specified, then the current interpreter abi tag is used

Reply:
This doesn't require a trusted host as there is no need for an external host when installing the Python Client. 

The process is as follows
- The environment running the test downloads the binary and dependencies into it's own environment.
- The test uploads the binary and dependencies to the Jupyter Notebook src directory, once it is running.
- The pip install targets the files already in Jupyter Notebook to install it without the need to goto a trusted host via the mirror.
- On completion of the test the binary and dependencies are deleted from both the Jupyter Notebook and the environment where the test is running.  They are never stored or pushed to any repository.

I also have the issue at the moment that I can't test using a trusted host on a disconnected cluster as Model Registry is not yet on RHOAI. It's also my understanding that you can't test on a disconnected cluster with ODH.  

I'm happy to change this over to what you are suggesting once a fully automated solution is available without the need for Manual intervention, for now though I prefer to have a test that will work anywhere without needing to manually set vars depending on the environment it is run.